lib/gaspar.rb

```diff
@@ -70,7 +70,7 @@ def initialize(source, target, pages, options)
       @source = source
       @target = target
       @pages = pages
-      @options = options
+      @options = options.empty? ? format_from_target(target) : options
```

Comment:
There are more options that we will include in next releases. Can you rewrite this line and verify that the **format** is not provided, instead of the entire **options** hash?  @GusGA 

```
Future options:
  --greyscale_threshold GREYSCALE_THRESHOLD
  -c CROP
  --line_length LINE_LENGTH
  --bitmap_resolution BITMAP_RESOLUTION
  -name NAME
  -pad PAD 
  -white WHITE
  -black BLACK
  -colmult COLMULT
  --whitespace {none,normalize,raw}
```